lib/Auth/Digest.php

```diff
@@ -133,7 +133,7 @@ protected function validate(): bool
                 return false;
             }
             // We need to add an md5 of the entire request body to the A2 part of the hash
-            $body = $this->request->getBody($asString = true);
+            $body = $this->request->getBody();
```

Comment:
`getBody()` is declared with no parameters, so passing a parameter here is meaningless - or do I miss something?